signalfx-org-metrics/README.md.jinja

```diff
@@ -11,43 +11,72 @@
+## USAGE
 
+Admins can view some of these metrics in built-in charts on the Organization Overview page. Like other metrics, you can display them in custom charts you create in order to monitor your system health.
 
-### About ByToken metrics
+### Metric pairs that provide total and ByToken values
 
 Some metrics send both a total value and a ByToken value, such as `sf.org.numAddDatapointCalls` and `sf.org.numAddDatapointCallsByToken`.
 
-The sum of all the ByToken values may be less than the value of the counterpart (non-token-based) metric; this is because no per-token value is sent for data that SignalFx retrieves on your behalf via AWS CloudWatch, GCP StackDriver, AppDynamics, or New Relic, because data sent from those integrations is not associated with a token. For example, if you sum the values sent for `sf.org.numAddDatapointCallsByToken`, the value may be less than the value of `sf.org.numAddDatapointCalls`, because the latter includes data from the specified integrations while the former does not.
+The sum of all the ByToken values for a metric may be less than the value of its non-token-based counterpart metric.
+The sums differ because Infrastructure Monitoring doesn't use a token to retrieve cloud service data you've requested with an integration.
+This applies to AWS CloudWatch, GCP StackDriver, AppDynamics, and New Relic.
+
+For example, if you sum the values for `sf.org.numAddDatapointCallsByToken`, the result may be less than the value of
+`sf.org.numAddDatapointCalls`, because `sf.org.numAddDatapointCalls` includes data from cloud service integrations,
+while `sf.org.numAddDatapointCallsByToken` does not.
+
+### Metrics with values for each metric type
+
+Some metrics have a value for each metric type (counter, cumulative counter or gauge), so you have three MTS per metric. Each MTS has a dimension named `category` with a value of `COUNTER`, `CUMULATIVE_COUNTER`, or `GAUGE`. Because you can have multiple MTS for these metrics, you need to use the `sum()` SignalFlow function to see the total value.
+
+For example, you might receive three MTS for `sf.org.numMetricTimeSeriesCreated`, one for the number of MTS that are counters, another for the number of MTS that are cumulative counters, and a third for the number of MTS that are gauges. To find the total number of MTS created, you need to sum those values.
+Use this SignalFlow expression:
+
+Also, you can filter by a single value of `category`, such as `GAUGE`, to see only the metrics of that type.
+
+### Metric that counts stopped detectors
 
+The metric `sf.org.numDetectorsAborted` monitors the number of detectors that Infrastructure Monitoring stopped because the detector reached a resource limit; this usually occurs when the number of MTS exceeds 250K. This condition also generates the event `sf.org.abortedDetectors`, which records details including the detector ID, the reason it stopped, and the value or limit of MTS or datapoints, whichever caused the detector to stop.
```

Comment:
250k or 250,000?

Reply:
250K is what was in the file originally.